virtualization/parameters.yml

```diff
@@ -3,9 +3,9 @@
 project_name: "presence"
 
 # Main hostname through which the vagrant box will be accessible
-hostname: "presence.lo"
+hostname: "presence.com"
```

Comment:
I'm guessing you used this because of oauth needing a proper name for redirecting on local development. The .test tld is imho the better choice there, as we have adopted it for other projects already as well. Check if google accepts that and then I'd advise changing to that.

Reply:
Yep, exactly. Unfortunately I just checked with .test and that didn't work. Google sez `Non-public domains not allowed: http://presence.test/auth/google/callback`.